StoreCgnExpirationActivity/table.ts

```diff
@@ -11,14 +11,17 @@ export const insertCgnExpiration = (
   cgnExpirationTableName: NonEmptyString
 ) => (
   fiscalCode: FiscalCode,
+  activationDate: Date,
   expirationDate: Date
 ): TaskEither<Error, TableService.EntityMetadata> =>
   taskify<Error, TableService.EntityMetadata>(cb =>
     tableService.insertOrReplaceEntity(
       cgnExpirationTableName,
       {
         PartitionKey: eg.String(date_fns.format(expirationDate, "yyyy-MM-dd")),
-        RowKey: eg.String(fiscalCode)
+        RowKey: eg.String(
```

Comment:
Why not using `RowKey=fiscalCode` and create two props for the other values?
es:
```
ActivationDate: activationDate,
ExpirationDate: expirationDate
```
[example](https://github.com/pagopa/io-functions-backoffice/blob/master/utils/audit_logs.ts#L48)
We can use query on table storage for date/fiscalCode